index.bs

```diff
@@ -4478,33 +4475,22 @@ entails in the ECMAScript language binding.
 <div class="example">
 
     Here is are some examples of wording to use to create and throw exceptions.
-    To throw a new [=simple exception=] named
-    {{TypeError}}:
+    To throw a new [=simple exception=] named <emu-val>TypeError</emu-val>:
 
     <blockquote>
-
-        [=Throw=] a TypeError.
-
+        [=Throw=] a <emu-val>TypeError</emu-val>.
```

Comment:
So thinking about this a little more, I'm wondering why we'd treat simple exceptions differently than we treat buffer source types. (That is, we have a 1:1 IDL representation of these objects and specs link to the WebIDL one.) To be clear, I'm not objecting to the idea of merging bindings and WebIDL were it make sense, but shouldn't we have a clearer overall picture before we do so?